components/Tabs/src/private/MDCItemBar.m

```diff
@@ -170,26 +170,32 @@ - (void)setItems:(NSArray<UITabBarItem *> *)items {
     [self stopObservingItems];
 
     _items = [items copy];
-    // Update the path to the selected item.
-    if (_lastSelectedIndexPath) {
-      if (_selectedItem) {
-        NSUInteger index = [_items indexOfObject:_selectedItem];
-        if (NSNotFound != index) {
-          _lastSelectedIndexPath = [NSIndexPath indexPathForItem:index inSection:0];
-        } else {
-          _lastSelectedIndexPath = nil;
-        }
+
+    // Determine new selected item.
+    UITabBarItem *newSelectedItem = nil;
+    if (_selectedItem) {
```

Comment:
I think these nested ifs could be simplified to:
if (_selectedItem && [_items containsObject:_selectedItem]) {
  newSelectedItem = _selectedItem;
} else {
  newSelectedItem = _items.firstObject;
}